pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -78,11 +78,11 @@ var (
 	OperatorVersionRHSSO               OperatorVersion = "8.0.1"
 	OperatorVersionRHSSOUser           OperatorVersion = "8.0.1"
 	OperatorVersionCodeReadyWorkspaces OperatorVersion = "2.0.0"
-	OperatorVersionFuse                OperatorVersion = "1.5.0"
+	OperatorVersionFuse                OperatorVersion = "7.5.0"
```

Comment:
Yeah, I'm a bit confused by this. I changed it because all of the other operator versions seem to make reference to the CSV version and not the actual operator image. 

I'm not sure which one is supposed to be used? 

cc @philbrookes

Reply:
It makes sense for me that this version should be the actual operator version and the product version to be defined in `Version<product>`. 

I don't think we have consistency across all products yet whether this csv version refers to the operator/product version. At the moment some of them refer to the product version and some the operator version.

Reply:
Ok, that makes sense, thanks Jameel

I'll change these now